README.md

```diff
@@ -81,38 +81,29 @@ cd pyopia-main
 
 For the next steps, you need to be located in the directory that contains the file 'environment.yml'.
 
-2. Create a virtual environment using the environment.yml (will create an environment called pyopia):
+1. (optional, but recommended) Create a virtual environment using the environment.yml. This will create an environment called pyopia, but with no dependencies installed. Dependencies are managed by poetry (in step 2):
```

Comment:
Poetry can also handle environment separation, it's one of its core features: https://python-poetry.org/docs/managing-environments/ 

Did you try without the conda env step, and just letting poetry do the environment creation?

https://python-poetry.org/docs/basic-usage/#activating-the-virtual-environment

Reply:
I was intending to keep this conda environment simple so that the people who are not already using poetry, but still using conda, don't need to do a system-wide poetry installation.